docs: add duration and last_played filters to Smart Playlist (#753)
music-assistant/music-assistant.io: src/content/docs/plugins/smart_playlist.md

- **Album type**: is / is not (filter by album type: album, single, EP, compilation, etc.)
- **Explicit**: filter tracks by explicit content marker (three options: allowed, explicit only, not allowed; provider-dependent)
- **Favorite**: yes (library mode)

> - **Year**: between from/to (you can also use only `from` or only `to`)

Reviewer: ```suggestion
- **Year**: between from/to (it is possible to use only `from` or only `to`)
```
Keeping the language third person passive

Author: Implemented

**Note:** Explicit content filtering depends on your music provider supplying explicit markers in their metadata. Not all music providers include this information, and availability varies by provider and region. If your provider does not supply explicit markers, this filter will have no effect.

Year examples:

- `from: 1969` means 1969 or newer.
- `to: 1980` means 1980 or older.

Duration examples:

> - `from: 0:30` means 30 seconds or longer (excludes very short tracks like interludes).

Reviewer: ```suggestion
- `from: 0:30` means 30 seconds or longer (excludes very short tracks such as interludes).
```

Author: Implemented

- `to: 5:00` means 5 minutes or shorter.
- `between: 3:00 and 6:00` means tracks between 3 and 6 minutes.

Last played examples:

> - `1 days ago` means tracks played within the last day.

Reviewer: Correct grammar would have this as “1 day ago”? Edit: I see your dropdown has `days` so understandable.

- `7 days ago` means tracks played within the last week.

> - Use this to create "recently played" playlists or to exclude tracks you've heard too recently.

Reviewer: ```suggestion
- Use this to create "recently played" playlists or to exclude tracks played recently.
```

Author: implemented
